core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerUtils.scala

```diff
@@ -17,10 +17,14 @@
 
 package org.apache.spark.scheduler.cluster.mesos
 
+import java.net.BindException
 import java.util.{List => JList}
 import java.util.concurrent.CountDownLatch
 
+import org.apache.spark.util.Utils._
+
 import scala.collection.JavaConverters._
```

Comment:
Check the Imports section of the Databricks style guide, these few lines need to be reordered.